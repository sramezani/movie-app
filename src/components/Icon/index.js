import React from 'react';
import PropTypes from 'prop-types';
import {
    StyleSheet,
} from 'react-native';

import Icon from 'react-native-vector-icons/dist/MaterialIcons';


/* Component ==================================== */
function IconView(props) {

    return (
        <>
            <Icon
                {...props}
            />
        </>
    );
}

/* Styles ==================================== */
const styles = StyleSheet.create({});

/* Component Props ==================================== */
IconView.propTypes = {};

IconView.defaultProps = {};

/* Export Component ==================================== */
export default IconView;
