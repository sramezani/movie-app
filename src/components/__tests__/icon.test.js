'use strict';

import 'react-native';
import React from 'react';
import Icon from '../Icon';
  
import renderer from 'react-test-renderer';

jest.setTimeout(15000);

test('renders correctly', () => {
  const tree = renderer.create(<Icon name="arrow-forward" size={30} color="#9a9" />).toJSON();
  expect(tree).toMatchSnapshot();
});
