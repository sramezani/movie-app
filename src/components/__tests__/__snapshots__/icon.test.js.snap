// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": "#9a9",
        "fontSize": 30,
      },
      undefined,
      Object {
        "fontFamily": "Material Icons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      Object {},
    ]
  }
>
  î—ˆ
</Text>
`;
